GitUI/CommandsDialogs/CommitDialog/FormCommitTemplateSettings.cs

```diff
@@ -43,6 +43,15 @@ private void LoadSettings()
                     _commitTemplates[i] = new CommitTemplateItem();
                 }
             }
+            else if (_commitTemplates.Length < _maxCommitTemplates)
+            {
```

Comment:
Add a comment that this is for migration

Reply:
done.